.env

```diff
@@ -0,0 +1,3 @@
+JWT_SECRET=IdoYotvat192837465
+ENCRYPTION_KEY=supercalifragelistic192837465
+CRYPTO_KEY=secretsecret123456789
```

Comment:
We never upload ENV files to git (for obvious reasons).
The .env should be in your git ignore